modules/styles.py

```diff
@@ -120,12 +126,16 @@ def apply_styles_to_prompt(self, prompt, styles):
     def apply_negative_styles_to_prompt(self, prompt, styles):
         return apply_styles_to_prompt(prompt, [self.styles.get(x, self.no_style).negative_prompt for x in styles])
 
-    def save_styles(self, style: Optional[PromptStyle] = None) -> None:
+    def save_styles(self, style: Union[PromptStyle, List[PromptStyle], None] = None) -> None:

```

Comment:
Use `|` to replace `Union`
Use `list` to replace `List`
`PromptStyle | list[PromptStyle] | None`

also remove the useless import